lms/envs/common.py

```diff
@@ -2067,6 +2067,9 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+############ CERTIFICATE VERIFICATION URL (STATIC FILES) ###########
+CERTIFICATES_STATIC_VERIFY_URL = "https://verify-test.edx.org/cert/"
```

Comment:
I'm a little bit concerned about this.  My understanding was that the previous implementation used the "download_url" from the `GeneratedCertificate` model, which was set by the response from the XQueue.  It looks like now we're hard-coding that URL.  This may turn out to be okay, assuming that all certificates have always used this URL, both on EdX.org and on OpenEdX installations.  We can check the edx.org case just by examining the database, but I'm less sure about the OpenEdX case.